assets/css/admin.css

```diff
@@ -147,4 +147,26 @@
 	left: 2px;
 	width: 23px;
 	height: 37px;
-}
\ No newline at end of file
+}
+
+#patreon_setup_screen {

```

Comment:
I think you also have this code in #79 -- is that intentional?

Reply:
I think 80 (this) and 81 respectively. I likely put them there due to a need to display the interface right during testing. We can remove them from one for consistency.